app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass()
+class Actor:
+    id: int
+    first_name: str
+    last_name: str

```

Comment:
The `Actor` class does not have an `objects` attribute, which is necessary for the `main.py` file to function correctly. Consider adding an `objects` attribute or modifying the approach to manage instances.